transmission_interface/src/simple_transmission_loader.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 PAL Robotics S.L.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "transmission_interface/simple_transmission_loader.hpp"
+
+#include <rclcpp/logging.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <pluginlib/class_list_macros.hpp>
+
+#include <memory>
+
+#include "hardware_interface/component_parser.hpp"
+#include "hardware_interface/hardware_info.hpp"
+#include "hardware_interface/types/hardware_interface_type_values.hpp"
+#include "transmission_interface/simple_transmission.hpp"
+#include "transmission_interface/transmission_interface_exception.hpp"
```

Comment:
Actually the header should be as follows:

1. `.hpp` file with the interfaces of implementation (own header)
2. system headers
3. external headers

Moreover, there is no reason to have CPP headers in `<>`.  This is how it is done in the rest of the repository. Therefore, propose this structure:

```suggestion
#include <memory>

#include "hardware_interface/component_parser.hpp"
#include "hardware_interface/hardware_info.hpp"
#include "hardware_interface/types/hardware_interface_type_values.hpp"
#include "pluginlib/class_list_macros.hpp"
#include "rclcpp/logging.hpp"
#include "rclcpp/rclcpp.hpp"
#include "transmission_interface/simple_transmission.hpp"
#include "transmission_interface/transmission_interface_exception.hpp"
```